README.md

```diff
@@ -18,10 +23,6 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```

Comment:
- [OPTIONAL] Please remove all unnecessary titles since they are empty  or add details to them (Prerequisites, run tests Deployment ....) 🚮